__tests__/app.test.js

```diff
@@ -185,4 +185,80 @@ describe('/api/articles', () => {
 
 });
 
+describe('/api/articles/:article_id', () => {
+    describe('PATCH on article_id', () => {
+
+        const updateArticle =
+        {
+            inc_votes: 23
+        }
+
+        test('/api/articles/:article_id,responds 200 with updated article  ', () => {
+
+
+
+            return request(app).patch('/api/articles/1').send(updateArticle).expect(200).then((response) => {
+
+                expect(response.body.article.length).toBe(1);
+
+                expect(response.body.article[0]).toEqual(expect.objectContaining({
```

Comment:
You will need to slightly adjust this, once you send back just the object, instead of array of one thing

Reply:
This as well:
 
return request(app).patch('/api/articles/1').send(updateArticle).expect(200).then((response) => {

                expect(response.body.article).toEqual(expect.objectContaining({
                    article_id: 1,
                    votes: 123,
                    title: expect.any(String),
                    body: expect.any(String),
                    created_at: expect.any(String),
                    author: expect.any(String),
                    topic: expect.any(String)
                }))

            }